eogrow/pipelines/import_tiff.py

```diff
@@ -41,7 +41,12 @@ def parameters_parser(cls, v: Tuple[ResizeParam, Tuple[float, float]]) -> Tuple[
 class ImportTiffPipeline(Pipeline):
     class Schema(Pipeline.Schema):
         output_folder_key: str = Field(description="The storage manager key of the output folder.")
-        tiff_folder_key: str = Field(description="The storage manager key of the folder containing the tiff to import.")
+        tiff_folder_key: Optional[str] = Field(
```

Comment:
Knowledge from the high wizards:

In the definition of the storage manager, these pre-defined folders are given as:
```python
PRESET_FOLDERS: Dict[str, str] = {"logs": "logs", "input_data": "input-data", "cache": "cache"}
```

You can set the default to `input-data` and get rid of almost all the other changes to the code :)

Reply:
done.